spec/models/activity_pub/actor_spec.cr

```diff
@@ -191,6 +201,24 @@ Spectator.describe ActivityPub::Actor do
       expect(actor.icon).to eq("icon link")
       expect(actor.image).to eq("image link")
       expect(actor.urls).to eq(["url link"])
+
+      expect(actor.attachments).not_to be_nil
+      attachments = actor.attachments.not_nil!
+      expect(attachments.size).to eq(2)
+      expect(attachments.all? { |a| a.type == "PropertyValue" }).to be_true
```

Comment:
once the attachment type is expanded, the value here will be "http://schema.org#PropertyValue" instead of just "PropertyValue"